### DISCUSSION TITLE
After Click on Navigate the data-toggle="collapse" Not Working

### DISCUSSION THREAD

**Discussion by RamiKhmer** [livewire/livewire#5984] [7. Bugs (archived)]
Hi, I have an issue when I try to click the collapse dropdown menu after clicking on the Livewire Navigate link, but the first load is it works, as usual, please help.

https://github.com/livewire/livewire/assets/54439421/56239b0a-2638-46e4-b7de-8feb4c4a229f

**Comment by JeffGepiga** [comment]
add data-navigate-once in your main script

just like below.
```
<script data-navigate-once  src="link_to_bootstrap_js" t></script>
```

**Comment by aftabkhaliq86** [comment]
or add you   bootstrap js in head
